### ISSUE TITLE
Unable to reproduce the code.

### ISSUE THREAD

**Issue by wazenmai** [VITA-Group/UVC#3]
Hello, I have read the paper and this is a great work on vision transformer compression! 

However, when I want to reproduce the code following the command on github. I always get 0 accuracy after joint training. **Could you provide the pre-trained model checkpoint so I can easily evaluate it on ImageNet?**

I simply followed the command on github for setting the environment and run the stage 1 uvc training command. I installed apex library without global option. I used 2 NVIDIA GeForce RTX 2080 GPU to run it.

<img width="695" alt="image" src="https://user-images.githubusercontent.com/51938055/210321434-d86b26e7-da90-4e8f-bc40-a039c18a54a5.png">

Best regards,
